### DISCUSSION TITLE
Fooocus Update Help

### DISCUSSION THREAD

**Discussion by Ceej1313** [lllyasviel/Fooocus#2465] [General]
Noooob Q:

My Fooocus doesn't update at the start. When I start, I get the following:

I:\Foocus\Fooocus_win64_2-1-831>.\python_embeded\python.exe -s Fooocus\entry_with_update.py
Update failed.
'object not found - no match for id (dc5b5238c83c63b4d7814ba210da074ddc341213)'
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py']

Then it starts as I first installed it. Anybody give me some pointers?

**Comment by mashb1t** [comment]
Please check it you have any changes with `git status` and if there are some check with `git diff <path>` if they are important.
To do a hard reset run:
```
git fetch origin
git reset --hard origin/main
```

Then check again with `git status` if you have changes and with `git log` if the latest commit is the latest one in https://github.com/lllyasviel/Fooocus/commits/main/.
